sdk/deviceprovisioningservices/azure-iot-deviceprovisioningservice/azure/iot/deviceprovisioningservice/__init__.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
```

Comment:
We discussed package rename azure-iot-deviceprovisioningservice -> azure-iot-deviceprovisioning